Makefile

```diff
@@ -21,6 +21,7 @@ endif
 TEST_CMD = $(DOCKER_COMPOSE_CMD) run publishing-e2e-tests bundle exec parallel_rspec -n $(TEST_PROCESSES) $(TEST_ARGS)
 
 all:
+	$(MAKE) stop
```

Comment:
Ugg, if you have no apps checked out that will fail won't it?

Reply:
If so we'll need to split out the clean into two steps.

1. clean_apps, clean_logs
2. clone
3. clean_docker
4. pull build